src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,26 @@
+const jwtService = require('../services/jwt.service.js');
+
+const authMiddleware = (req, res) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
+
+  if (!userData) {
+    res.sendStatus(401);
+
+    return;
```

Comment:
After sending a response with `res.sendStatus(401)`, you should call `return next();` to properly terminate the middleware function.